jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/utils/PayloadCreator.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.homeoffice.digital.sas.jparest.utils;
+
+import java.util.List;
+
+public class PayloadCreator {
+
+    public static String createIDPayload(List<String> fieldNameList, List<String> fieldValueList){
```

Comment:
Regarding the Types used here, we could improve the conciseness of the calling code by make the field **values** of a generic type that extends Serializable. 
Then the calling code can pass in UUIDs and numeric values in their original type without having to call `.toString()`
In conjunction with the map comment, it might look something like this:
`public static <S extends Serializable> String createPayload(Map<String, S> dataMap)`

Reply:
Done

Reply:
nice, just the other numeric types are still being passed in as a string, see new comment for where it can be be passed in as it's corresponding field's type.